base/multidimensional.jl

```diff
@@ -776,6 +776,34 @@ end
     end
 end
 
+# manually written-out specializations for 1 and 2 arguments to save compile time
+@inline function _unsafe_getindex!(dest::AbstractArray, src::AbstractArray, I_1::Union{Real, AbstractArray})
```

Comment:
Can we just make the body here a macro and use that three times. I'm just worried that we'll end up changing one of these and forgetting about the others.